spec/factories/news.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :news do
+    name { "MyString" }
+    content { "MyString" }
```

Comment:
You could use [Faker](https://github.com/faker-ruby/faker) in this case to improve your factories.